spec/models/author_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Book, type: :model do
```

Comment:
Please separate model spec files. This file contains tests for `Book`, `Author`, and has extraneous code/text/